### PR TITLE
SMP DQM codes updated after fixing the bug 

### PR DESCRIPTION
**Pull request by gwalia**

### CODE REVIEW

File: DQM/Physics/src/EwkMuDQM.cc [hunk 1]
```diff
@@ -307,30 +381,46 @@ void EwkMuDQM::bookHistograms(DQMStore::IBooker & ibooker,
...
+{
+   using namespace edm;
   Handle<View<Muon> > muonCollection;
-  if (!ev.getByToken(muonTag_, muonCollection)) {
-    // LogWarning("") << ">>> Muon collection does not exist !!!";
-    return;
-  }
-  unsigned int muonCollectionSize = muonCollection->size();
+  if (!iEvent.getByToken(muonTag_, muonCollection)) {
+     std::cout  << ">>> Muon collection does not exist !!!" << std::endl;
```
Comment (dmitrijus) [inline DQM/Physics/src/EwkMuDQM.cc]: std::cout is thread unsafe!
Please go back to using LogWarning.

**Review by dmitrijus — CHANGES_REQUESTED** [review]

**Comment by dmitrijus** [comment]
Please run, `clang-format -i --style=Google` on the cpp files and review the changes you have made.
It is barely readable: lots of if-statements don't have properly matching brackets.

**Comment by gwalia** [comment]
thanks for the comments, we will update the code with the changes suggested.

**Comment by dmitrijus** [comment]
It is still messy and barely readable code.
If you do a clang format - it should go as a separate commit, before applying your own changes.

**Comment by fabozzi** [comment]
Hi Genius, is it possible to move on with this PR?

**Comment by gwalia** [comment]
Hi,
   There were few indentation changes suggested by Dmitrijus Bugelskis and we made a new pull request (PR #16322) with the updated codes.

Thanks,
Genius
________________________________

**Comment by dmitrijus** [comment]
please close this PR

**Pull request closed without merging**